.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,20 @@
 ## Description
 
 _Please add a description of the changes that this PR introduces and the files that
-are the most critical to review._ 
+are the most critical to review._
 
 If this PR is non-trivial/large/complex, please ensure that you have either
 created an issue that the team's had a chance to respond to, or had some
 discussion with the team prior to submitting substantial pull requests. The team
-can be reached via GitHub Discussions or the Cosmos Network Discord server in
-the #cometbft channel. GitHub Discussions is preferred over Discord as it
-allows us to keep track of conversations topically.
-https://github.com/cometbft/cometbft/discussions
+can be reached via GitHub Discussions:
+https://github.com/celestiaorg/celestia-core/discussions
```

Comment:
@evan-forbes Please let me know if there are other communication mediums that should be included.

Reply:
makes sense to me